### ISSUE TITLE
Fix pip dependency issues

### ISSUE THREAD

**Issue by lennartbrandin** [painebenjamin/app.enfugue.ai#23]
When trying to install with pip ive run into the following issues:
```
Collecting enfugue[tensorrt]
  Using cached enfugue-0.1.0.tar.gz (1.1 MB)
  Preparing metadata (setup.py) ... done
ERROR: Cannot install enfugue[tensorrt]==0.1.0 and enfugue[tensorrt]==0.1.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    enfugue[tensorrt] 0.1.1 depends on polygraphy<0.48 and >=0.47
    enfugue[tensorrt] 0.1.0 depends on polygraphy<0.48 and >=0.47

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```
Which can be resolved as mentioned in this [reddit comment](https://www.reddit.com/r/StableDiffusion/comments/14kx6zu/comment/jpx63nf/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
(`pip install enfugue --extra-index-url https://pypi.ngc.nvidia.com`)

Further on I've encountered another dependency issue:
```
ERROR: Cannot install enfugue==0.1.0 and enfugue==0.1.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    enfugue 0.1.1 depends on latent-diffusion
    enfugue 0.1.0 depends on latent-diffusion

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

**Comment by ChadDa3mon** [comment]
I would suspect that's your issue. If you're not using some sort of virtual envirornment (pyenv, conda etc) you'll eventually run into issues like this.

**Comment by painebenjamin** [comment]
This has been resolved as of 0.1.2!

**Comment by d8ahazard** [comment]
This appears to be an issue again with the latest version:

<img width="741" alt="image" src="https://github.com/painebenjamin/app.enfugue.ai/assets/1633844/ffc9007a-e0ea-4c39-b6fd-0630b7a1246c">
